File: src/main/java/io/rsocket/demo/StreamAdapter.java
package io.rsocket.demo;

import org.springframework.social.twitter.api.StreamDeleteEvent;
import org.springframework.social.twitter.api.StreamListener;
import org.springframework.social.twitter.api.StreamWarningEvent;
import org.springframework.social.twitter.api.Tweet;

public class StreamAdapter implements StreamListener {
  @Override public void onTweet(Tweet tweet) {
  }

  @Override public void onDelete(StreamDeleteEvent deleteEvent) {
  }

  @Override public void onLimit(int numberOfLimitedTweets) {
  }

  @Override public void onWarning(StreamWarningEvent warningEvent) {
  }
}

File: src/main/java/io/rsocket/demo/StreamAdapter.kt
package io.rsocket.demo

import org.springframework.social.twitter.api.StreamDeleteEvent
import org.springframework.social.twitter.api.StreamListener
import org.springframework.social.twitter.api.StreamWarningEvent
import org.springframework.social.twitter.api.Tweet

open class StreamAdapter : StreamListener {
  override fun onTweet(tweet: Tweet) {}

  override fun onDelete(deleteEvent: StreamDeleteEvent) {}

  override fun onLimit(numberOfLimitedTweets: Int) {}

  override fun onWarning(warningEvent: StreamWarningEvent) {}
}
